sympy/physics/control/lti.py

```diff
@@ -891,6 +896,15 @@ def __truediv__(self, other):
     def __neg__(self):
         return Series(TransferFunction(-1, 1, self.var), self)
 
+    def _to_expr(self):
+        temp = []
+        for arg in self.args:
+            if isinstance(arg, TransferFunction):
+                temp.append(arg.to_expr())
+            else:
+                temp.append(arg._to_expr())
```

Comment:
This block seems to be repeated in a few places. Why don't all classes just have a method of the same name?

Reply:
I thought that the `to_expr` method for `Series` and `Parallel` objects might not have any application for the end-user. However, in the `TransferFunction` class, @namannimmo10 demonstrated a few use cases (https://github.com/sympy/sympy/pull/21634#discussion_r654944089) where it was genuinely helpful for the user.

Reply:
The methods should just have the same name for all classes so you don't have to check `isinstance` like this